frappe/printing/page/print/print.js

```diff
@@ -742,18 +712,14 @@ frappe.ui.form.PrintView = class {
 	}
 
 	refresh_print_options() {
-		this.print_formats = frappe.meta.get_print_formats(this.frm.doctype);
-		const print_format_select_val = this.print_sel.val();
-		this.print_sel
-			.empty()
-			.add_options([
-				this.get_default_option_for_select(__("Select Print Format")),
-				...this.print_formats,
-			]);
-		return (
-			this.print_formats.includes(print_format_select_val) &&
-			this.print_sel.val(print_format_select_val)
-		);
+		if (
+			frappe.meta.get_print_formats(this.frm.doctype).includes(this.print_sel.val()) ||
```

Comment:
Yes! When a valid format is already selected, we don't want to overwrite it with some default. You can try this: create two formats, set one as default, select the other one in print view, click on "..." > "Customize", then immediately navigate back to the print view. With this condition, it keeps the original value. Without it, it would reset to the default print format.